src/Collage.elm

```diff
@@ -359,7 +361,36 @@ Scaling by 2 doubles both dimensions and quadruples the area.
 -}
 scale : Float -> Collage msg -> Collage msg
 scale s collage =
-    { collage | scale = collage.scale * s }
+    scaleXY ( s, s ) collage
+
+
+{-| Scale a collage horizontally (in its local space) by a given factor.
+
+Scaling by 2 doubles the width.
```

Comment:
Maybe add "and doubles the area" to match `scale`? Same holds for the documentation of `scaleY`.